applications/PfemFluidDynamicsApplication/custom_strategies/strategies/two_step_v_p_strategy.h

```diff
@@ -211,7 +211,6 @@ namespace Kratos
 
     double Solve() override
```

Comment:
Just for you to know as I see that you are trying to clean up the strategy. The `Solve` method shouldn't be no longer used. Instead the `InitializeSolutionStep`, `SolveSolutionStep` ... should be employed.

Reply:
Thank you Ruben, we will update it too, also in the other strategies.
In the meantime, I merge this pull request as it solves a bug..